Runtimes/Core/Demangling/CMakeLists.txt

```diff
@@ -9,27 +9,26 @@ add_library(swiftDemangling STATIC
   "${SwiftCore_SWIFTC_SOURCE_DIR}/lib/Demangling/Remangler.cpp"
   "${SwiftCore_SWIFTC_SOURCE_DIR}/lib/Demangling/NodeDumper.cpp"
   "${SwiftCore_SWIFTC_SOURCE_DIR}/lib/Demangling/Errors.cpp")
-target_compile_definitions(swiftDemangling PRIVATE swiftCore_EXPORTS)
+target_compile_definitions(swiftDemangling PRIVATE swiftCore_EXPORTS
+  -DSWIFT_SUPPORT_OLD_MANGLING=$<BOOL:${SwiftCore_ENABLE_OBJC_INTEROP}>
+  $<$<BOOL:${SwiftCore_ENABLE_TYPE_PRINTING}>:-DSWIFT_STDLIB_HAS_TYPE_PRINTING>
+  -DSWIFT_HAVE_CRASHREPORTERCLIENT=$<BOOL:${SwiftCore_ENABLE_CRASH_REPORTER_CLIENT}>)
 
 target_include_directories(swiftDemangling
-  PRIVATE "${PROJECT_SOURCE_DIR}/../../include")
+  PRIVATE
+    "${PROJECT_SOURCE_DIR}/../../include"
```

Comment:
Is this not `${SwiftCore_SWIFTC_SOURCE_DIR}/include`?

Reply:
Missed it.